src/user/how_to_get_help.rst

```diff
@@ -16,24 +16,22 @@ you to the proper documentation and/or help you via the chat.
 
 GitHub issues
 -------------
-You can open issues at `staged-recipes<https://github.com/conda-forge/staged-recipes/issues>`_
+You can open issues at `staged-recipes <https://github.com/conda-forge/staged-recipes/issues>`__
 to request a new package or to report a problem with staged-recipes itself (our port of entry to conda-forge).
 
 Or you can open an issue about a specific package at the package feedstock via
 ``https://github.com/conda-forge/<PACKAGE-NAME>-feedstock/issues``
 
 When opening issues be sure to:
 
-1. Try a new environment first following conda-forge `install instructions<https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge`_.
-2. Always open a new one when an issue is closed.
-In the packaging world symptoms may be similar but the causes are usually very different.
-3. Fill in the required information. Without the output of `conda list` and `conda info -a`
-the team cannot debug the problem.
+1. Try a new environment first following conda-forge `install instructions <https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge>`__.
+2. Always open a new one when an issue is closed. In the packaging world symptoms may be similar but the causes are usually very different.
+3. Fill in the required information. Without the output of `conda list` and `conda info -a` the team cannot debug the problem.
 
 
-You can open issues at `conda-forge.github.io <https://github.com/conda-forge/conda-forge.github.io/issues>`_
+You can open issues at `conda-forge.github.io <https://github.com/conda-forge/conda-forge.github.io/issues>`__
 for long threads involving infrastructural, architectural or ecosystem wide discussions.
-Please note that some of these discussions will be turned into policy via the `CFEP process <https://github.com/conda-forge/conda-forge-enhancement-proposals>`_.
+Please note that some of these discussions will be turned into policy via the `CFEP process <https://github.com/conda-forge/conda-forge-enhancement-proposals>`__.
```

Comment:
Why do these have 2 trailing underscores?

Reply:
One underscore is a named reference, two are anonymous references. IIRC, for named references  the name has to be unique. I generally used anonymous references throughout the documentation to avoid having to guarantee uniqueness of the names.

Reply:
Ooooh OK!